### PR TITLE
rails-blog-app: models

### PR DESCRIPTION
**Pull request by omarbabou**
- [ ] Create model classes for all entities as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
 - [ ]  Set up associations between models.
 - [ ] Insert data with ['rails c' command](https://guides.rubyonrails.org/command_line.html#bin-rails-console).
 - [ ] Create some custom methods in the models
 - [ ] Create a method called `current_user` in the ApplicationController.

### CODE REVIEW

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @omarbabou,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] Please make sure that your project has the Postgres database set up not the Sqlite as requested in the project requirements.

> **Note:** you can add `gem 'pg', '~> 1.1'` in your gemfile. Perhaps it will solve the issue.

https://github.com/omarbabou/rails-blog-app/blob/65723da23e0dfdfdac7bb204e7f3df3ef1691277/Gemfile#L11-L14

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: db/migrate/20220618102159_create_users.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ def change
         t.string :name
         t.string :photo
         t.text :bio
+        t.integer :posts_counter, default:0
         t.integer :posts_counter
```
Comment (youmari) [inline db/migrate/20220618102159_create_users.rb]: - [x] you have duplicated column `posts_counter` in this migration which leads to an error when migrating to a database so please try to remove one and leave only the first one in order to migrate without a problem

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @omarbabou , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @omarbabou , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**